nailgun/entities.py

```diff
@@ -1096,6 +1135,23 @@ def __init__(self, server_config=None, **kwargs):
         }
         super(ContentView, self).__init__(server_config, **kwargs)
 
+    def read(self, entity=None, attrs=None, ignore=()):
+        """Fetch an attribute missing from the server's response.
+
+        For more information, see `Bugzilla #1237257
+        <https://bugzilla.redhat.com/show_bug.cgi?id=1237257>`_.
+
+        """
+        if (getattr(self._server_config, 'version', parse_version('6.1')) <
```

Comment:
Ugh. The handling of version information in NailGun is one thing I am *not* happy with. `getattr` is used because `self._server_config.version` is not guaranteed to exist.

This statement can also be written as such:

```python
version = getattr(self._server_config, 'version', parse_version('6.1'))
if version < parse_version('6.1'):
    pass  # some workaround
```

This code can be read as, "if we're talking to a Satellite system older than 6.1, perform a workaround".